### PR TITLE
Customize Adyen drop-in component

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Customize Adyen drop-in component

Add Adyen environment variables to the example.

Add payment configuration hook to simplify module.

Add button translations.

Add CSS styles to override Adyen styles.

Add mocks for payment providers query.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #471** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/471" target="_blank">Graphite</a> 
    * **PR #476** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/476" target="_blank">Graphite</a> 
      * **PR #479** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/479" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/479?utm_source=stack-comment).

File: apps/store/src/lib/l10n/locales.ts [hunk 1]
```diff
@@ -6,38 +6,45 @@ export type LocaleData = {
   path: Locale
   locale: Locale
   language: Language
+  currencyLocale: 'en-SE' | 'sv-SE' | 'en-NO' | 'nb-NO' | 'en-DK' | 'da-DK'
```
Comment (alebedev) [inline apps/store/src/lib/l10n/locales.ts:9]: Removed that in merged PR, may need some conflict resolution

File: apps/store/src/services/adyen/AdyenCheckout.tsx [hunk 2]
```diff
@@ -60,26 +69,17 @@ const useAdyenConfiguration = () => {
...
-            },
-            error: {
-              color: theme.colors.gray700,
-            },
-          },
-        },
+      locale: localeToAdyenLocale(locale),
+      translations: {
+        'no-NO': { payButton: payButtonText },
+        'da-DK': { payButton: payButtonText },
```
Comment (alebedev) [inline apps/store/src/services/adyen/AdyenCheckout.tsx:76]: Can there be a case when we need to provide different translations here?  We can pass specific locale to `t()` to translate into specific language

**Review by alebedev — APPROVED** [review]
💰💰💰

File: apps/store/src/lib/l10n/locales.ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/lib/l10n/locales.ts:9]: I already rebased on top of your changes :)

File: apps/store/src/services/adyen/AdyenCheckout.tsx (hunk 2 above)
Comment (robinandeer) [reply apps/store/src/services/adyen/AdyenCheckout.tsx:76]: hmm don't think there's such a need, this is taken from web-onboarding so it's worked before :)

File: apps/store/src/lib/l10n/locales.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/lib/l10n/locales.ts:9]: Odd, I removed currencyLocale completely in favor of locale

**Pull request merged**